tests/test_import_cli.py

```diff
@@ -407,3 +407,107 @@ def test_skip_validation(patched_prompt):
     # Remove created db
     if os.path.exists(db_path):
         os.remove(db_path)
+
+
+class TestImportWithMissingDBTableSQLite(unittest.TestCase):
+    def setUp(self):
+        ds = DataStore("", "", "", 0, "cli_import_test.db", db_type="sqlite")
+        ds.initialise()
+
+    def tearDown(self):
+        os.remove("cli_import_test.db")
+
+    @patch("pepys_import.utils.error_handling.custom_print_formatted_text", side_effect=side_effect)
+    def test_import_with_missing_db_table(self, patched_print):
+        conn = sqlite3.connect("cli_import_test.db")
+        load_spatialite(conn, None)
+
+        # We want to DROP a column from the States table, but SQLite doesn't support this
+        # so we drop the table and create a new table instead
+        conn.execute("DROP TABLE Geometries;")
+        conn.close()
+
+        temp_output = StringIO()
+        with redirect_stdout(temp_output):
+            process(path=DATA_PATH, archive=False, db="cli_import_test.db", resolver="default")
+        output = temp_output.getvalue()
+
+        print(output)
```

Comment:
@robintw - is printing the output to console just needed during development?  I see we test it in code in subsequent lines, so suspect we don't need to do it here or at line 498.

Reply:
I'm currently debugging a test failure that only seems to occur on the CI - hence adding the print statement. They will be deleted once I've sorted the failure.

Reply:
Is it now ready to go @robintw ?

Reply:
I've just pushed a commit to remove those print statements as it now works on CI. Should be ready to merge now.